dataherald/finetuning/openai_finetuning.py

```diff
@@ -43,8 +43,24 @@ def __init__(self, storage: Any, fine_tuning_model: Finetuning):
         )
         self.client = OpenAI(api_key=db_connection.decrypt_api_key())
 
+    @staticmethod
+    def map_finetuning_status(status: str) -> FineTuningStatus:  # noqa: PLR0911
```

Comment:
Instead of having many if condition we could use a dict.
Something like this:
```
mapped_statuses = {
 "queued": FineTuningStatus.QUEUED.value,
 "running": FineTuningStatus.RUNNING.value,
 .....
}
if status not in mapped_statuses:
    return FineTuningStatus.QUEUED
return mapped_statuses[status] 
```